addon/ng2/blueprints/ng2/files/src/client/app/__name__.component.spec.ts

```diff
@@ -1,4 +1,4 @@
-import {describe, it, expect, beforeEachProviders, inject} from 'angular2/testing';
+import {describe, it, expect, beforeEachProviders, inject} from '@angular/tesing';
```

Comment:
Actually, not right now... https://www.npmjs.com/package/@igorminar/tesing

Working on it.